tuxemon/event/actions/random_monster.py

```diff
@@ -80,3 +79,4 @@ def start(self) -> None:
         current_monster.current_hp = current_monster.hp
 
         trainer.add_monster(current_monster)
+        trainer.tuxepedia[monster_slug] = SeenStatus.caught.value
```

Comment:
`SeenStatus.caught`

please do this for all cases of SeenStatus

Reply:
Exactly what I'm doing